xcell/mappers/mapper_2MPZ.py

```diff
@@ -20,6 +19,7 @@ def __init__(self, config):
         """
         self._get_defaults(config)
         self.z_edges = config.get('z_edges', [0, 0.5])
+        self._get_coords(config)
```

Comment:
Since you're not getting anything, maybe call it `load_coords` or, even better, make `_get_coords` return `ra_name, dec_name` and in this line you do `self.ra_name, self.dec_name = self._get_coords()`

Reply:
done